### PR TITLE
feature/new-user-Dto

### PR DESCRIPTION
**Pull request by cristi6891**

### CODE REVIEW

File: src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+import java.time.LocalDate;
+@Getter
+@Setter
+
 public class UserDTO {
+
+    public String firstName;
+    public String lastName;
+    public String email;
+    public String username;
```
Comment (Miller1995) [inline src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java]: - @cristi6891  use camelCase for naming of all fields;

File: src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java [hunk 2]
```diff
@@ -1,4 +1,22 @@
 package com.SocialMedia.SocialMedia.models.dto;
 
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+@Getter
+@Setter
+
```
Comment (Miller1995) [inline src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java]: - @cristi6891 maybe we need to use @Data from Lombok (for All arguments and Null arguments Constructor, toString etc). 
- refactor code, remove free line (exem line 9 and line 21 etc)

File: src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java [hunk 3]
```diff
@@ -1,4 +1,22 @@
...
 public class UserDTO {
+
+    public String firstName;
```
Comment (Miller1995) [inline src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java]: - @cristi6891 maybe we need use fundamental OOP concepts Encapsulation (can't be accessible outside the class) ?  
More info:
 https://stackoverflow.com/questions/48820066/why-do-we-declare-private-variables-in-java

File: src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java (hunk 1 above)
Comment (cristi6891) [reply src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java]: in this case there is no need for camelCase

File: src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java (hunk 2 above)
Comment (cristi6891) [reply src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java]: changed @Getter and @Setter with @Data
refactor done - removed free lines;

File: src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java (hunk 3 above)
Comment (cristi6891) [reply src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java]: changed from public to private

File: src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java (hunk 2 above)
Comment (Miller1995) [reply src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java]: ok

File: src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java (hunk 3 above)
Comment (Miller1995) [reply src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java]: ok, good job

**Pull request merged**